packages/fake-browser/src/apis/storage.ts

```diff
@@ -90,6 +91,10 @@ const managedStorage = {
   ...defineStorageArea('managed'),
   QUOTA_BYTES: 5242880 as const,
 };
+const sessionStorage = {
+  ...defineStorageArea('session'),
+  QUOTA_BYTES: 5242880 as const,
```

Comment:
From https://developer.chrome.com/docs/extensions/reference/storage/#property-session

```suggestion
  QUOTA_BYTES: 10485760 as const,
```